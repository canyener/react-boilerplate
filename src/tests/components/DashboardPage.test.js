import React from 'react'
import { shallow } from 'enzyme'

import DashboardPage from '../../components/DashboardPage'

describe('ExpenseDashboardPage', () => {
  describe('#Snapshots', () => {
    test('Should render ExpenseDashboard correctly', () => {
      const wrapper = shallow(<DashboardPage />)
      expect(wrapper).toMatchSnapshot()
    })
  })
})

