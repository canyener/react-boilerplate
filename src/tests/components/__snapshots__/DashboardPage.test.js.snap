// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpenseDashboardPage #Snapshots Should render ExpenseDashboard correctly 1`] = `
<div>
  Dashboard page content
</div>
`;
